### PR TITLE
fix: id:44544 Changes in table styling

### PR DESCRIPTION
**Pull request by Farhan-emgage**
Changes Done for Table styling

### CODE REVIEW

File: src/styles/foundation/colors.scss [hunk 1]
```diff
@@ -94,6 +94,13 @@ $color-palette-data: (
     dark: #202e78,
     darker: #000639,
     text: #3e4155
+  ),
+  grey:(
+    base: #212121,
+    darkest:#424242,
+    darker:#616161,
+    light:#9e9e9e,
+    reverse:#fff
```
[the remark this answers is not in the document]
Comment (weegrom) [reply src/styles/foundation/colors.scss:103]: @Farhan-emgage please take care of the request to remove the "grey" mixin and use existing colors asap so we can get the PR merged.

[the remark this answers is not in the document]
Comment (weegrom) [reply src/styles/foundation/colors.scss:103]: @Farhan-emgage please resolve so we can get this merged.

[the remark this answers is not in the document]
Comment (Farhan-emgage) [reply src/styles/foundation/colors.scss:103]: @weegrom For that we have to discuss about it in meeting

File: themes/Delicious/foundation/spacing.scss [hunk 2]
```diff
@@ -3,6 +3,7 @@ $unit: 4px;
 $spacing-data: (
   extra-tight: rem($unit),
   tight: rem($unit * 2),
+  tighter: rem($unit * 3),
   base: rem($unit * 4),
   loose: rem($unit * 5),
   extra-loose: rem($unit * 8)
```
[the remark this answers is not in the document]
Comment (weegrom) [reply themes/Delicious/foundation/spacing.scss:9]: @Farhan-emgage this change still needs to be made. You made it to the base styles, but not to Delicious theme.

[the remark this answers is not in the document]
Comment (weegrom) [reply themes/Delicious/foundation/spacing.scss:9]: @Farhan-emgage nevermind, I see the change in another PR.

**Pull request merged**